nuimo/src/main/kotlin/com/senic/nuimo/NuimoBluetoothController.kt

```diff
@@ -58,6 +61,13 @@ class NuimoBluetoothController(bluetoothDevice: BluetoothDevice, context: Contex
         reset()
     }
 
+    fun rebootToDfuMode() {
```

Comment:
Return value is convenient because then you can call `rebootToDfuMode()` without checking beforehand for `supportsRebootToDfuMode`.